src/helix-ui/elements/HXAccordionElement.js

```diff
@@ -27,12 +27,17 @@ export class HXAccordionElement extends HXElement {
 
     $onConnect () {
         this.$upgradeProperty('currentPanel');
+        // FIXME: panels are not connected at this point
```

Comment:
Untrue. Children _should_ be present, but Edge/IE require the `$onConnect` to be deferred to the next event loop using `setTimeout(this.$onConnect, 0)` in `HXElement.js`.